ui/components/multichain/import-token-link/import-token-link.tsx

```diff
@@ -20,7 +20,10 @@ import {
   getIsTokenDetectionInactiveOnMainnet,
 } from '../../../selectors';
 
-export const ImportTokenLink = ({ className, ...props }) => {
+export const ImportTokenLink: React.FC<{ className: string }> = ({
```

Comment:
In the component library all types for a component are kept in a `component-name.types.ts` file. I realize that this component only has a single prop but I don't think this would scale well for other components. It might be worth discussing some conventions when it comes to organizing types on the wallet UX team. We probably want to standardize this across the extension so I would suggest moving this to a `import-token-link.types.ts` file and having something like

```
export interface ImportTokenLinkProps {
  /**
   * Additional class name for the ImportTokenLink component.
   */
  className?: string;
}
```

Reply:
Nice shout! Updated here https://github.com/MetaMask/metamask-extension/pull/20474/commits/08cbbc9ab93f60cc0f6f7670b2575726fe01e6ad